packages/medusa/src/api/middlewares/error-handler.ts

```diff
@@ -57,9 +57,11 @@ export default () => {
       case MedusaError.Types.INVALID_ARGUMENT:
         break
       default:
-        errObj.code = "unknown_error"
-        errObj.message = "An unknown error occurred."
-        errObj.type = "unknown_error"
+        if (process.env.NODE_ENV !== "development") {
```

Comment:
question: Should we check startsWith('dev') to give more flexibility on env naming?